README.md

```diff
@@ -1,4 +1,44 @@
 cerberus
 ========
 
-Automatic [alert system](http://vitillo.github.io/cerberus/dashboard/) for telemetry histograms.
+Automatic [alert system](http://mozilla.github.io/cerberus/dashboard/) for [telemetry](http://telemetry.mozilla.org/) histograms.
+
+Cerberus detects changes in the distribution of histograms (histogram regressions), and posts alerts for these to the [medusa](https://github.com/mozilla/medusa) alert service.
+
+Cerberus is also responsible for sending out email reminders for histograms that are expiring in upcoming versions.
+
+The production instance is currently hosted on [telemetry-regression-alerts](ec2-52-13-120-199.us-west-2.compute.amazonaws.com) on the us-east-1 AWS region. All the production files live in `/mnt/telemetry/cerberus/`. There is a cronjob 
```

Comment:
Please remove this paragraph. The deployment is about to be properly automated and all informations pertaining it will reside in the repository.